index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Front Page</title>
+</head>
+<body>
+     
+    <section class="admin-panel hide-from-mobile">
+        <img class="social-icons" src="images/fb-icon.png" alt="">
+        <img class="social-icons" src="images/twitter-icon.png" alt="">
+        <p class="social-panel">English</p>
+        <p class="social-panel">My page</p>
+        <p class="social-panel">Logout</p>
+    </section>
+    <header class="header-container">
+        <img class="header-bg" src="images/head-bg.jpg" alt="">
+        <div id="hamburger" onclick="showMobileMenu()">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>
+        <div id="close-icon" onclick="closeMobileMenu()">X</div>
+        <nav id="mobile-menu">
+                <a class="nav-item" href="about.html">About</a>
+                <a class="nav-item" href="index.html">Program</a>
+                <a class="nav-item" href="">Join</a>
+                <a class="nav-item" href="">Sponsor</a>
+                <a class="nav-item" href="">News</a>
+                <a class="nav-item" href="">CC Campaign</a>
+        </nav>
+
+        <nav class="desktop-menu hide-from-mobile">
+            <a class="nav-item-desk" href="about.html">About</a>
+            <a class="nav-item-desk" href="index.html">Program</a>
+            <a class="nav-item-desk" href="">Join</a>
+            <a class="nav-item-desk" href="">Sponsor</a>
+            <a class="nav-item-desk" href="">News</a>
+            <a class="nav-item-desk" href="">CC Campaign</a>
+        </nav>
+        <div><h2 class="logo hide-from-mobile">Logo</h2></div>
```

Comment:
- [ ] Your logo should be an icon, not text, please remember that is this what you will be presenting to recruiters, I suggest you make it so attractive and professional so that recruiters can reach you out :wink:  :upside_down_face: 
![home page](https://user-images.githubusercontent.com/88894525/204042584-0328710f-0300-4a88-b8a9-7ea490e0e23c.png)

Reply:
This is not a requirement, for now, the text is just a placeholder, I will change it (along with lorem texts) when I'm actually applying for jobs!!

Reply:
- [ ] [optional] @bill7pearl great work on your  project, however, the navbar background color doesn't match the design given, kindly ensure it matches it as this is a project requirement 
![image](https://user-images.githubusercontent.com/61408860/204055388-6442bbdf-fcf8-413a-b1ef-d14027022096.png)

Reply:
- [ ] [optional] In mobile view the hamburger button is on the right instead of the left as it's in the design, kindly use images instead of text, this is making your work design to differ from the actual design intention.